exploring-ship-logbooks/word_count.py

```diff
@@ -0,0 +1,52 @@
+import pandas as pd
+import math
+
+def word_count(data, columns, key_words):
```

Comment:
I was thinking we would go through and count the words that have to do with slave (slave, slaves... and more words). There's an example word set that we can expand upon on kaggle. I can quickly make a second version of this script to count all words included in the logbooks.